app/main.py

```diff
@@ -1,2 +1,90 @@
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.length = 0
+        self.hash_table: list[None | tuple[Hashable, hash, Any]] = [None] * 8
+        self.load_factor = 2 / 3
+        self.capacity = 2
+
+    def __len__(self) -> int:
+        return self.length
+
+    @staticmethod
+    def add_item_to_table(
+            table: list[None | tuple[Hashable, hash, Any]],
+            key: Hashable,
+            value: Any
+    ) -> None:
+        hash_ = hash(key)
+        index = hash_ % (len(table) - 1)
```

Comment:
The hash calculation uses `len(table) - 1`, which might not distribute keys uniformly. Consider using `len(table)` instead.